app/graphql/enums/filter_match_enum.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Enums::FilterMatchEnum < Types::BaseEnum
```

Comment:
Thank you for going above and beyond the minimum requirements for HYACINTH-582 with all of these filters!  It's great having so many types.  If (and only if) it would be easy to do, would you be okay with throwing 'STARTS_WITH' in here too, since that's the only HYC2 one that's missing at this point?